kak-lsp.toml

```diff
@@ -6,163 +6,184 @@
 # set to 0 to disable
 timeout = 1800 # seconds = 30 minutes
 
-[language.bash]
+# This section overrides language IDs.
+# By default, kak-lsp use filetypes for the IDs.
+# See https://microsoft.github.io/language-server-protocol/specifications/specification-current/#textDocumentItem
+[language_ids]
+# javascript = "javascriptreact"
+protobuf = "proto"
+sh = "shellscript"
+# typescript = "typescriptreact"
+
+[language_server.bash-language-server]
 filetypes = ["sh"]
 roots = [".git", ".hg"]
 command = "bash-language-server"
 args = ["start"]
 
-[language.c_cpp]
+[language_server.clangd]
```

Comment:
we used to send `c_cpp` always. Maybe we should keep doing that because I'm not sure if all *.h files are detected correctly.